src/02_spec/07_modules/dem_uart/index.rst

```diff
@@ -0,0 +1,29 @@
+UART Device Emulation Module (DEM_UART)
+=======================================
+
+.. figure:: img/osd_dem_uart.*
+   :alt: Device Emulation Module UART
+
+The UART Device Emulation Module (DEM_UART) connects to the bus of a given CPU on one side,
+and to the Debug Interconnect on the other.
+Towards the CPU it behaves like a UART-16550 device, but instead of transmitting information
+over a UART-Interface it instead passes it to a host PC via the Debug Interconnect Interface.
+
+The DEM_UART_16550 component is responsible for emulating the behavior of a UART chip, while
```

Comment:
This alludes to the internal structure of the module? This structure it not relevant for this documentation, only the behavior at its boundaries is.